.github/workflows/chromatic.yml

```
@@ -1,10 +1,18 @@
 name: "Chromatic"
 
-on: push
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
 
 jobs:
   chromatic:
     runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'chromatic') }}
```

```suggestion
    if: ${{ github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'chromatic') }}
```

I'm guessing otherwise it still won't actually run on merge to `main`.

---

Oh, yep, thank you!

I'm also wondering why it didn't trigger the chromatic run when I added the tag, because it seems like when the deploy tags are added, it runs them immediately, but chromatic didn't run till the next push. 

Maybe it has something to do with it being a deploy to an environment?

---

I was wondering the same thing!

I think [this](https://github.com/mbta/skate/pull/2368/files#r1457945443) is the fix. Otherwise the workflow doesn't interpret labeling as an event worthy of rerunning (or considering whether to rerun).

---

That seems to have fixed it, Thanks!